README.md

```diff
@@ -10,47 +10,99 @@ master|[![Build Status](https://mseng.visualstudio.com/AzureDevOps/_apis/build/s
-- [ ] Windows installer
-- [ ] Improved GitHub support (platform-native UI)
-- [ ] BitBucket
-- [ ] Linux installer
+- [ ] 32-bit Windows support (only 64-bit Windows is supported currently)
+- [ ] Linux support
+- [ ] macOS/Linux native UI
 
 ## Download and Install
 
-To use GCM Core, you can download the [latest installer](https://github.com/Microsoft/Git-Credential-Manager-Core/releases/latest) for your platform. To install, double-click installation package and follow the instructions presented.
+### macOS Homebrew
 
-### Git Credential Manager for Mac and Linux (Java-based GCM)
+The preferred installation mechanism is using Homebrew; we offer a Cask in our custom Tap.
+
+To install, run the following:
+
+```shell
+brew tap microsoft/git
+brew cask install git-credential-manager-core
+```
+
+#### Git Credential Manager for Mac and Linux (Java-based GCM)
 
 If you have an existing installation of the 'Java GCM' on macOS and you have installed this using Homebrew, this installation will be unlinked (`brew unlink git-credential-manager`) when GCM Core is installed.
 
+#### Uninstall
+
+To uninstall, run the following:
+
+```shell
+brew cask uninstall git-credential-manager-core
+```
+
+---
+
+### macOS Package
+
+We also provide a [.pkg installer](https://github.com/Microsoft/Git-Credential-Manager-Core/releases/latest) with each release. To install, double-click the installation package and follow the instructions presented.
+
+#### Uninstall
+
+To uninstall, run the following:
+
+```shell
+sudo /usr/local/share/gcm-core/uninstall.sh
+```
+
+---
+
+### Windows
+
+You can download the [latest installer](https://github.com/Microsoft/Git-Credential-Manager-Core/releases/latest) for Windows. To install, double-click the installation package and follow the instructions presented.
+
+#### Git Credential Manager for Windows
+
+GCM Core installs side-by-side any existing Git Credential Manager for Windows installation and will take precendence over the older GCM. GCM Core will use any existing credentials generated by GCM for Windows so you shouldn't need to re-authenticate.
+
+#### Uninstall (Windows 10)
+
+To uninstall, open the Settings app and navigate to the Apps section. Select "Git Credential Manager Core" and click "Uninstall".
+
+#### Uninstall (Windows 7-8.1)
+
+To uninstall, open Control Panel and navigate to the Programs and Features screen. Select "Git Credential Manager Core" and click "Remove".
+
 ## How to use
 
-You don't! GCM gets called when Git needs credentials for an operation. For example, when pushing (`git push`) to [Azure DevOps](https://dev.azure.com), it automatically opens a window and initializes an OAuth2 flow to get your personal access token.
+You don't! Git Credential Manager Core gets called when Git needs credentials for an operation. For example, when pushing (`git push`) to [Azure DevOps](https://dev.azure.com), it automatically opens a window and initializes an OAuth2 flow to get your personal access token.
```

Comment:
Maybe we can tone down the "You don't" while we're at it? Maybe start with "Git Credential Manager Core is called implicitly by Git, when so configured, it is not intended to be called directly by the user."?

Reply:
Updated.